Update admin-settings.php

maymond opened the pull request:
PHP 5.4 requires a variable to be used for the empty() function, but the get_option from WordPress returns a value which causes the settings page to return blank. Switched to using a boolean false check on the get_option function returned value.

maymond committed:
Update admin-settings.php

PHP 5.4 requires a variable to be used for the empty() function, but the get_option from WordPress returns a value which causes the settings page to return blank. Switched to using a boolean false check on the get_option function returned value.

ionux commented:
Nice catch, @maymond!  We'll run a brief test on our test server internally and merge if everything looks good.  Thanks!

maymond commented:
Not a problem, spent longer than I would like to admin tracking that one down so I'm happy to save others the trouble.

aleitner commented:
@maymond It looks like your logic is a bit off. 
If the get_option('bitpayToken') returns false then the pairing form should be visible, because there is no token. 

Please change to `(false === get_option('bitpayToken'))`, and I'll happily merge your pull request

maymond committed:
Update admin-settings.php

Fixed logic per request so it properly displays the form if the token is not there.

maymond commented:
That makes sense, and would explain the odd behavior I saw on the site I am testing the plugin on. Updated the commit. For reference I also noticed the same call in the payment class file.

aleitner commented:
Perfect, Thanks

ionux commented:
@maymond thanks again for the contribution and feel free to submit any future PRs to this or any of our other projects.  I'm always appreciative of the assistance third-party developers like yourself lend my team!  If you send me your mailing address (rich@bitpay.com), I'll be happy to send you a couple BitPay stickers as a token of thanks. :)
